packages/@aws-cdk/aws-servicediscovery/lib/namespace.ts

```diff
@@ -39,28 +39,6 @@ export interface BaseNamespaceProps {
   readonly description?: string;
 }
 
-export interface NamespaceAttributes {
```

Comment:
Why are you deleting this? Better to add the importer then if it was missing, no?

Reply:
I did not want to ponder whether importing a namespace is actually useful or not. And if the importer is just `fromArn`, then the attributes are still useless.